tests/test_env.py

```diff
@@ -100,7 +99,9 @@ def install(self, requirements):
 
 
 @pytest.mark.isolated
-@pytest.mark.skipif(sys.version_info > (3, 6, 5), reason='inapplicable')
-def test_default_pip_is_upgraded_on_python_3_6_5_and_below():
+def test_default_pip_is_never_too_old():
     with build.env.IsolatedEnvBuilder() as env:
-        assert not subprocess.check_output([env.executable, '-m', 'pip', '-V']).startswith(b'pip 9.0')
+        version = subprocess.check_output([env.executable, '-c', 'import pip; print(pip.__version__)']).strip()
```

Comment:
you don't have to do this dance if you pass universal_newlines=True

Reply:
Good, I was going to do encoding=, but that was 3.6+.